src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,134 @@
 package core.basesyntax;
```

Comment:
I'll approve it since PR was reviewed several times and major things was fixed, but there are many unnecessary empty lines.